pynet/datasets/height_bb.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
```

Comment:
I suppose you mean rename height_bb.py in ukb.py. I will do that : 

1. apply modifications corresponding to you remarks
2. commit them
3. rename height_bb in ukb

(hope this will work)